Answers.md

```diff
@@ -0,0 +1,16 @@
+1. What is the difference between an adaptive website and a fully responsive website?
+A responsive website will resize with every pixel width change, whereas an adaptive website has
+fixed designs for each specific screen size and won't change until that pixel ratio is reached. 
+
+2. Describe what it means to be mobile first vs desktop first.
+A mobile first design is designed specifically for mobile screens. You would use "min-width:" 
+in your @media. 
+
+3. What does `font-size: 62.5%` in the `html` tag do for us when using `rem` units? 
+It makes it so that 1rem is 10px. This isn't a rule to follow, it just makes using rem easier.
+
```

Comment:
Good! I like that you answer the questions! that's just as important as the challenge in understanding it!